bootstrap/file/file.go

```diff
@@ -0,0 +1,64 @@
+package file
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/edgexfoundry/go-mod-bootstrap/v3/bootstrap/interfaces"
+)
+
+func LoadFile(path string, timeout time.Duration, provider interfaces.SecretProvider) ([]byte, error) {
+	var fileBytes []byte
+	var err error
+
+	parsedUrl, err := url.Parse(path)
+	if parsedUrl.Scheme == "http" || parsedUrl.Scheme == "https" {
```

Comment:
err is never checked. It will be non-nil if local file path used. 
```suggestion
	if err != nil && (parsedUrl.Scheme == "http" || parsedUrl.Scheme == "https") {
```